### ISSUE TITLE
Any chance of an update?

### ISSUE THREAD

**Issue by smulder** [timbowhite/botbouncer#2]
I am trying to build / install for node 9.11.1 and eventually would love to be able to install for 10, but there are depreciation errors. Any chance the dependencies or other requirements could be updated?

Thanks tons! This looks pretty promising for me.

**Comment by timbowhite** [comment]
Yeah you were probably having problems with the sqlite3 module, the version that botbouncer was using was very outdated.

I've updated the dependencies, I was able to get it installed and most tests working on node v10.6.0.

Please note that the Bitcoin payments checking isn't going to work since the blockr.io API isn't around anymore.

**Comment by smulder** [comment]
Thanks for the quick turnaround! Super appreciated.
